src/context/reducers/dataReducer.js

```diff
@@ -5,6 +5,12 @@ import {
   historyConstants,
   playlistConstants,
 } from "../actions/dataActions";
+import { tokenName } from "../providers/AuthProvider";
+
+const isLoggedIn = () => {
+  if (window.localStorage.getItem(tokenName)) return true;
+  else return false;
```

Comment:
Is accessing the localStorage through window necessary

Reply:
no, can be omitted if there is no other variables with same name.

Reply:
You have another variable named localStorage?

Reply:
no, it can be removed 👍🏼